android-json-form-wizard/src/test/java/com/vijay/jsonwizard/widgets/ToasterNotesFactoryTest.java

```diff
@@ -18,164 +12,66 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.Mock;
-import org.mockito.Mockito;
 import org.mockito.MockitoAnnotations;
+import org.robolectric.Robolectric;
 
 import java.util.List;
+import java.util.Set;
 
 public class ToasterNotesFactoryTest extends BaseTest {
```

Comment:
extend `FactoryTest` instead, remove the `JsonFormActivity` declaration and assignment call super on `setUp`